portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts

```diff
@@ -52,9 +51,9 @@ export const download = (
       const rspHeader = req.getResponseHeader("Content-Disposition");
 
       let filename = "download";
-
       if (rspHeader) {
-        filename = rspHeader.split('"')[1];
+        let rspHeaderDecoded = decodeURIComponent(rspHeader);
```

Comment:
```suggestion
        let rspHeaderDecoded = decodeFileName(rspHeader);
```

Please use the `decodeFileName` function located at `portal-ui/src/common/utils.ts`, that function will handle the error when a string cannot be decoded

Reply:
Tried that that doesn't work